aci/resource_aci_fabricnodeidentp.go

```diff
@@ -134,6 +148,26 @@ func setFabricNodeMemberAttributes(fabricNodeIdentP *models.FabricNodeMember, d
 	return d, nil
 }
 
+func getAndsetDecommissionedNodes(client *client.Client, serial string, d *schema.ResourceData) (*schema.ResourceData, error) {
+
+	// Get node data and verify if the node is attached to the switch.
+	dn := fmt.Sprintf("client-[%s]", d.Get("serial").(string))
+	dhcpClientCont, err := client.Get(dn)
+	if err != nil {
+		if !strings.Contains(err.Error(), "Error retrieving Object: Object may not exist") {
```

Comment:
Since this is an old resource and in this case. Previously when the object didn't exists ->  in the debug you can still see these lines, as it exits at this particular case but doesn't error out. In this case if i this is the exact error i want to continue and not exit the code.

Reply:
Okay, great. Makes sense.